rusk/tests/common/state.rs

```diff
@@ -96,11 +96,15 @@ pub fn generator_procedure(
         )));
     }
 
+    let dummy_voters_pubkey =
+        missed_generators.iter().map(|pk| (pk.clone(), 1)).collect();
+
```

Comment:
If you use the missed_generator key as voter it results in having all the slash test fail

Reply:
Replaced with None